### DISCUSSION TITLE
I submitted a bug report for Maui and it got added to XamarinCommunityToolkit

### DISCUSSION THREAD

**Discussion by brightertools** [CommunityToolkit/Maui#948] [General]
I followed the link to create a bug report and it added it to 

https://github.com/xamarin/XamarinCommunityToolkit/issues/1946

I initially thought I added it in the wrong place, but the link goes there and not here?

Why is this?

**Comment by bijington** [comment]
> I followed the link to create a bug report and it added it to 
> 
> 
> 
> https://github.com/xamarin/XamarinCommunityToolkit/issues/1946
> 
> 
> 
> I initially thought I added it in the wrong place, but the link goes there and not here?
> 
> 
> 
> Why is this?
> 
> 
> 
> 

Sorry it looks like we have managed to break our bug report template. There should be an option to raise a bug here, the one you were presented with was for MauiCompat which is a special build of the Xamarin Community Toolkit

**Reply by brightertools** [reply]
ah ok, should I re-submit the bug? and where?

**Reply by bijington** [reply]
The template has now been fixed. If you would be able to raise it under this repository that would be a huge help. Sadly I don't think we can transfer issues between that repository and this one. Thanks

**Comment by CliffAgius** [comment]
Closed and answered
